pkg/kritis/review/review_test.go

```diff
@@ -552,43 +606,45 @@ func TestGetAttestationAuthoritiesForISP(t *testing.T) {
 		Auths: authMock,
 	})
 	tcs := []struct {
-		name        string
-		aList       []string
-		shouldErr   bool
-		expectedLen int
+		name      string
+		aName     string
+		shouldErr bool
+		returnNil bool
```

Comment:
Maybe reverse this to be `returnAttAuth`? To make it clearer when reading

Reply:
done. reversed to `returnAuths`